website/extensions/horizontal-rule.md

```diff
@@ -0,0 +1,29 @@
+# Horizontal Rule
+
+The `horizontalRule` node is used to represent a thematic break in the document, typically rendered as a horizontal line.
+
+<!-- @include: @/examples/horizontalRule.md -->
+
+## Usage
+
+```ts
+import { defineHorizontalRule } from 'prosekit/extensions/horizontalRule'
+
+const extension = defineHorizontalRule()
+```
+
+## Commands
+
+### insertHorizontalRule
+
+```ts
+editor.commands.insertHorizontalRule()
+```
+
+## Input Rules
+
+Input three hyphens (---), asterisks (\*\*\* followed by a space or a line break), or underscores (\_\_\_ followed by a space or a line break) on an empty line , and it will automatically convert to a `horizontalRule` node.
```

Comment:
Is there a reason why you chose different behaviors (i.e. not needing to press space or enter after typing `---`)?

Reply:
Although [commonmark](https://spec.commonmark.org/0.31.2/#thematic-break) supports three syntaxes for `<hr>`, I'd like to keep it simpler by only have one syntax (`---`). Both Notion and Craft only support `---`, and other text editors like MS Word, Google Docs and Apple Notes don't even support any of them. Having three syntaxes could confuse non-markdown users in my humble opinion.

Reply:
Perhaps we should follow Notion’s approach.